tests/integration/cli/up_test.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import busybox_single_app_bundle_fixture
+from dusty.systems.docker import get_docker_env
+
+class TestUpCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestUpCLI, self).setUp()
+        busybox_single_app_bundle_fixture(num_bundles=2)
+        self.run_command('bundles activate busyboxa busyboxb')
+
+    def test_basic_up_command(self):
+        run_output = self.run_command('up')
+        self.assertTrue('Your local environment is now started!' in run_output)
+
+        docker_output = subprocess.check_output(['sh', '-c', "docker ps"], env=get_docker_env())
```

Comment:
Also this seems like it should be a general assert function provided by the test case. Could probably even piggyback on the existing inspect method to do it easily.